tests/main/xdg-open/task.yaml

```diff
@@ -85,11 +84,6 @@ execute: |
     ensure_xdg_open_output "help:snapcraft"
     ensure_xdg_open_output "apt:snapcraft"
     ensure_xdg_open_output "zoommtg://snapcraft.io"
-    ensure_xdg_open_output "slack://5NAPPY111/magic-login/bcaf81ee-07b1-4362-9c09-ff46bd6e1bb9"
-    ensure_xdg_open_output "msteams://snapcraft.io"
-
-    # Ensure XDG_DATA_DIRS was prefixed with snap specific location
-    test "$(tail -1 /tmp/xdg-open-output)" = "XDG_DATA_DIRS=$SNAP_MOUNT_DIR/test-snapd-desktop/current/usr/share:/usr/share"
```

Comment:
is this diff right?

Reply:
it is not :( thank you

Reply:
I pushed an update but please double check, this test has changed quite a bit